curbs/README.md

```diff
@@ -366,16 +366,17 @@ is due to priorities: a `loading` rule that is higher priority than a `no loadin
-- `no unloading` - no unloading allowed; implies that parking is also prohibited
-- `stopping` - stopping briefly to pick up or drop off passengers
-- `no stopping` - stopping, loading, unloading, and parking are all prohibited; not a typical travel lane
-- `travel` - represents curbside lanes intended for moving vehicles, like bus lanes, bike lanes,
-  and rush-hour-only travel lanes; implies that parking, loading, unloading, and stopping are prohibited.
+| Activity     | Description                                                                                                         | parking allowed | loading allowed | unloading allowed | stopping allowed | travel allowed |
+|--------------|---------------------------------------------------------------------------------------------------------------------|-----------------|-----------------|-------------------|------------------|----------------|
+| parking      | pay stop and leave vehicle unattended                                                                               | ✅               | ✅               | ✅                 | ✅                | ?              |
+| no parking   | may not stop and leave vehicle unattended                                                                           | ❌               | ✅               | ✅                 | ✅                | ?              |
+| loading      | loading of goods                                                                                                    | ❌               | ✅               | ?                 | ✅                | ?              |
+| no loading   | no loading allowed                                                                                                  | ❌               | ❌               | ?                 | ✅                | ?              |
+| unloading    | unloading of goods                                                                                                  | ❌               | ?               | ✅                 | ✅                | ?              |
+| no unloading | no unloading allowed                                                                                                | ❌               | ?               | ❌                 | ✅                | ?              |
+| stopping     | stopping briefly to pick up or drop off passengers                                                                  | ❌               | ❌               | ❌                 | ✅                | ?              |
+| no stopping  | not a typical travel lane                                                                                           | ❌               | ❌               | ❌                 | ❌                | ✅              |
+| travel       | represents curbside lanes intended for moving vehicles, like bus lanes, bike lanes, and rush-hour-only travel lanes | ❌               | ❌               | ❌                 | ❌                | ✅              |
```

Comment:
@schnuerle , this does a good job of representing my problem w/ no stopping vs travel. In this model, they represent the same thing.

Reply:
Love the table, nice work! For the "?" you could use the Emoji White Question Mark or even Wavy Dash to show it's not clearly defined.

In my mind, these activities that are used for defining rules for curb policy, no stopping and travel are different, beyond what the table currently shows, maybe more about physical build.  An example "travel" area by the curb could be a flush curb to the travel lane of a road, while a no stopping rule could apply to a curb cut in an area that is never a normal travel lane. I'm not sure of the policy differences and needs between these rule definitions of 'travel' vs 'no stopping' - maybe you just want to be clear that this zone at this time of day is a travel lane and be clear about that, vs just a no stopping area?

Reply:
I've replaced the question mark w/ the emoji

Reply:
>while a no stopping rule could apply to a curb cut in an area that is never a normal travel lane.

If you can't stop there and you can't travel there, I'm not sure what you could do there. I can think of one example of such a situation, but it suggests we ought to have a `no travel` activity instead of a `travel` one. It also bends the definition of a curb: 

A promenade or large sidewalk-like area could have a `no travel` activity defined for all non-municipal car-like classes and a `no parking` policy defined for scooters.

Reply:
It seems that temporary uses that block use of the curb, such parklets for outdoor dining, or a construction lay down area could represent an opportunity to use the proposed `no travel` activity, as only using `no stopping` could communicate that the curb lane could still be utilized in some fashion as a travel lane.

Reply:
Also, local jurisdictions will need to do some translation work to fit local regulatory practices to CDS.  As an example, in SF we use `no stopping` on signage to designate curb space that is temporarily permitted for moving trucks to stop and conduct `loading` activity.